### PR TITLE
Mobile first Portfolio page project pull request

### PR DESCRIPTION
**Pull request by Habtamu210**
In this PR, I have completed the following activities:
- Added header and headline section of the site
- Applied flex to layout elements on the page
- Added images and Icons
- configured linter for HTML and CSS

### CODE REVIEW

File: port.css [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  color: #fff;
+  position: relative;
+  bottom: 40px;
+}
+
+p {
+  margin-left: 1rem;
+  width: 566px;
+  height: 123px;
+  font-family: 'Poppins', sans-serif;
```
Comment (Ugboaja-Uchechi) [inline port.css:69]: - [ ] The paragraph text overflows. Kindly fix this so it looks as close as possible to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: port.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+#back {
+  position: relative;
+  bottom: 100px;
+  right: 100px;
+  height: 298.3157177564727px;
+  width: 300.5108468215952px;
+  left: 900.43310546875px;
+  top: -10px;
+  border-radius: 0;
+}
```
Comment (Ugboaja-Uchechi) [inline port.css:123]: - [ ] The background image doesn't display on the mobile view only on the desktop view. Kindly fix this 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Habtamu210 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 

## Required Changes ♻️

- [ ] Great job making your `README` descriptive in the `main` pull request 👍 . Kindly make sure your `README` is also descriptive in all pull requests 🙏 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+#back {
+  position: relative;
+  bottom: 100px;
+  right: 100px;
+  height: 298.3157177564727px;
+  width: 300.5108468215952px;
+  left: 900.43310546875px;
+  top: -10px;
+  border-radius: 0;
+}
```
Comment (Alfredbis29) [inline index.css:117]: - [ OPTIONAL]  I can see that you made some changes but you still need to improve on your design as required in the previous review 
after making changes this how your design looks like :point_down: 
toooop2

Figma Design: 
gotPddd

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Roboto:ital,wght@1,900&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Roboto:ital,wght@1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="main">
+    <nav class="logo">
+      <h1 class="me">Habtamu</h1>
+      <img class="vec" src="images/Union.png" alt="">
+    </nav>
+    <section class="initial">
```
Comment (Alfredbis29) [inline index.html:21]: - [ OPTIONAL] The container of the section is a header tag, kindly change the `div` tag to `header` tag :upside_down_face: 
Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
> # _Status: Required Changes_ 
> Hi @mukund816,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
 * No linter errors. :heavy_check_mark:
 * Good README.  :heavy_check_mark:
 * Correct GitHub Flow.  :heavy_check_mark:
 * Good commit messages.  :heavy_check_mark:
 

 
 ## Required Changes 
 _Check the comments under the review._
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
 
I wish you the best in making these changes and in your subsequent projects. heart

File: index.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        <li><a href="#"><img src="images/github.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/Linkedin.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/H.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
+        <li><a href="#"><img src="images/M.svg" alt=""></a></li>
+      </ul>
+    
+   <!--  <div class="bg">
+      <img id="back" src="images/Illustration-header.png">
+    </div> -->
```
Comment (Kamba56) [inline index.html:40]: - [Optional] Please remove all commented code from your work. This will make your code neat.

**Review by Kamba56 — APPROVED** [review]
Hi @Habtamu210 ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏👏

## Highlights
- [x] All linters passed
- [ ] Great use of Github flow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**